TwitchLib.Api.Helix/Webhooks.cs

```diff
@@ -14,6 +15,30 @@ public Webhooks(IApiSettings settings, IRateLimiter rateLimiter, IHttpCallHandle
         {
         }
 
+        #region GetWebhookSubscriptions
+        public Task<GetWebhookSubscriptionsResponse> GetWebhookSubscriptionsAsync(string after = null, int first = 20, string accessToken = null)
+        {
+            if (first < 1 || first > 100)
+                throw new BadParameterException("'first' must between 1 (inclusive) and 100 (inclusive).");
+
+            var getParams = new List<KeyValuePair<string, string>>
+                {
+                    new KeyValuePair<string, string>("first", first.ToString())
+                };
+            if (after != null)
+                getParams.Add(new KeyValuePair<string, string>("after", after));
+
+
+            accessToken = GetAccessToken(accessToken);
+
+            if (string.IsNullOrEmpty(accessToken))
+                throw new BadParameterException("'accessToken' be supplied, set AccessToken in settings or ClientId and Secret in settings");
```

Comment:
Use nameof(accessToken) instead of 'accessToken'.
Use the constructor which has a parameter name, and pass along nameof(accessToken). --> BadParameterException("SomeDescription", nameof(accessToken))

Gives clearer feedback and is more refactor proof.